src/Tickets/Emails/Admin/Preview_Data.php

```diff
@@ -220,7 +220,7 @@ public static function get_attendees( $args = [] ): array {
 	 */
 	public static function get_tickets( $args = [] ): array {
 		$default = [
-			[
+			(object) [
```

Comment:
You're treating the preview data as an array of objects where the [actual data](https://github.com/the-events-calendar/event-tickets/blob/f897806088bf5a9fbd597102da2785667841a60f/src/Tickets/Emails/Legacy.php#L102) that's being delivered in the email is an array of arrays. 

These changes are adjusted for tests but not for real emails, these changes will break the Tickets and RSVP emails.